stories/demos/CommonFeatures.tsx

```diff
@@ -1,12 +1,13 @@
-import { useState, useCallback, useMemo } from 'react';
+import { useState, useMemo } from 'react';
 import { css } from '@linaria/core';
 import faker from 'faker';
 
 import DataGrid, { SelectColumn, TextEditor, SelectCellFormatter } from '../../src';
-import type { Column, SortDirection } from '../../src';
+import type { Column } from '../../src';
```

Comment:
`SortColumn` is already exported at the index:
```suggestion
import type { Column, SortColumn } from '../../src';
```